css/styles.css

```diff
@@ -101,3 +105,69 @@ li div.text-box {
 .edit-icon:hover {
   cursor: pointer;
 }
+
+.delete-button {
+  color: #dc2626;
+  font-size: 14px;
+  font-family: sans-serif;
+  text-decoration: underline;
+}
+
+.hide-delete {
+  visibility: hidden;
+}
+
+.delete-button:hover {
+  cursor: pointer;
+}
+
+.modal {
+  position: fixed;
+  width: 320px;
+  height: 130px;
+  background-color: white;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -60%);
```

Comment:
You probably want this to be `(-50%, -50%)`. The transform function moves the origin point for the element, and two -50% values sets the origin as the center point of the element.